prov/efa/src/rxr/rxr_pkt_type_misc.c

```diff
@@ -646,8 +646,7 @@ void rxr_pkt_handle_receipt_recv(struct rxr_ep *ep,
 		return;
 	}
 
-	tx_entry->rxr_flags |= RXR_RECEIPT_RECEIVED;
-	if (!(tx_entry->bytes_acked > 0)) {
+	if (!(tx_entry->rxr_flags & RXR_LONG_MSG)) {
```

Comment:
Please adjust the sequence of the block to:

```
if (tx_entry->rxr_flags & RXR_LONG_MSG) {
} else {
}
```
Negative condition is harder to understand.

Reply:
Comment addressed, resolving.